news/_posts/2020-03-12-ps17-minor-release-lifecycle.md

```diff
@@ -0,0 +1,126 @@
+
+By the end of this week, if no new regression has been reported, the RC1 becomes the final Release and **PrestaShop 1.7.7.0 is out !**
+
+... However, I dont think it ever happened than RC1 was flawless.
+
+Most of the time, some more regressions will be reported. When a new regression is reported, we fix it and we reset the timer. So there is a RC2, and if more regressions are reported, there is a RC3... until no new regressions are reported.
+
+Finally the latest built Release Candidate becomes the stable PrestaShop 1.7.7.0 software release.
+
+# Summary
+
+- The cycle begins when we start working on a minor version scope. When we complete the scope of this version, we trigger the Feature Freeze.
+
+Expected duration: 4 months
+
+- Following Feature Freeze an Alpha build is delivered and tested by QA team. QA team will then report all known defects and we will be able to plan then deliver a Beta build.
+
+Expected duration: 1 month
+
+- The Beta Build is open to everybody and we host a one-month long period for both maintainers and community to test the build.
+
+Expected duration: 1 month
+
+- At the end of the Beta period, we fix all the bugs that have been found and maintainers deliver a Release Candidate 1 build.
+After a week, if no bugs have been reported in RC1 build, we deliver the stable release. However if bugs a reported, we fix them and deliver a RC2, and so on until we build a version with no reported issues. This last Release Candidate becomes the stable release.
+
+Expected duration: from 1 week (if RC1 is flawless) to 1 month, possibly 2 months
+
+### Calendar
+
+If you sum all expected phases duration, you reach 6 months. This is why we expect to deliver 2 minor releases by year. 2 phases of development, 2 phases of feature freeze and 2 beta periods.
+
+# Some more details
+
+This lifecycle has been adopted by PrestaShop team in 2018 following issues we had in the release process of PrestaShop 1.7.3 and 1.7.4. It seems to have proven its qualities as PrestaShop 1.7.5 and 1.7.6 releases were a lot more robust. However this comes with a cost: all of this process requires a huge investment from the maintainers, the QA team, the Product team ... and the community ! Also the stable release involves a lot of people of the PrestaShop crew: marketing, partnerships, training, everybody participates in the launch, be it marketing material or communication. Launching a minor version is a huge work for all the PrestaShop company and community, which is why we cannot do it too often.
+
+Another sidenote: we are very often asked "when will next minor version be out ?"
+As you can understand from this process, there is no such date. The answer is "Version 1.7.X will be out when we deliver a Release Candidate that nobody reports bug for". This can take one week or three months.
```

Comment:
```suggestion
Another side note: people often ask "when will next minor version be out?"
As you can understand from this process, there is cannot be a defined date. The answer is "Version 1.7.X will be out when a Release Candidate is delivered with no known regressions". This can take one week or three months.

However, the teams have been exploring ways of providing better visibility to the community regarding the project's development.
```